neo/Wallets/test_VerificationContract.py

```diff
@@ -0,0 +1,82 @@
+import binascii
+from io import BytesIO
+from neo.Utils.NeoTestCase import NeoTestCase
+from neo.IO.MemoryStream import StreamManager
+from neo.IO.BinaryReader import BinaryReader
+from neo.IO.BinaryWriter import BinaryWriter
+from neo.Wallets.VerificationContract import VerificationContract
+from neo.SmartContract.ContractParameterType import ContractParameterType
+from neo.Wallets.KeyPair import KeyPair
+
+
+class VerificationContractTest(NeoTestCase):
+    # Captured data from C# client 2.4.1.0
+    raw_verification_contract = binascii.unhexlify(
+        '27fd098f9ffc3309fbc310cccfde9a9fb445a8790100232102103a7f7dd016558597f7960d27c516a4394fd968b9e65155eb4b013e4040406eac')
+    expected_address = 'AWLYWXB8C9Lt1nHdDZJnC5cpYJjgRDLk17'
+    expected_isStandard = True
+    # should we make this a proper ENUM like C#?
```

Comment:
Need an opinion on the ContractParameter type. It's currently a class, but I think a proper enum could do as well.